### PR TITLE
Create SignIn-page

### LINKED ISSUE #30: Create SignUpPage
As a new user
I need to be able to create an caaount
therefore I need a sign up form.
Screenshot 2021-09-13 at 13 23 40

### PR DESCRIPTION
**Pull request by aronvaupel**
Create SignIn-page
Improve styling of some elements (minor changes, don´t have to be checked)

closes #30

### CODE REVIEW

File: src/app/pages/SignInPage/SignInPage.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            value="Submit"
+            onClick={handleSubmit}
+            type="submit"
+            disabled={
+              !validUsername ||
+              !validPhonenumber ||
+              !validEmail ||
+              !validPassword
+                ? true
+                : false
```
Comment (marwinburesch) [inline src/app/pages/SignInPage/SignInPage.tsx:82]: I'd have this up top underneath those functions as a variable

File: src/app/components/ActionButton/ActionButton.tsx [hunk 2]
```diff
@@ -3,20 +3,23 @@ import styles from './ActionButton.module.css';
 
 export type ActionButtonProps = {
   value: 'Submit' | 'Upload photo';
-  onClick: () => void;
+  onClick?: () => void;
   type?: 'submit' | 'button';
   isActive?: boolean;
+  disabled: boolean;
```
Comment (marwinburesch) [inline src/app/components/ActionButton/ActionButton.tsx:9]: I think this should be optional

File: src/app/pages/SignInPage/SignInPage.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  return (
+    <div className={styles.wrapper}>
+      <Header />
+      <main className={styles.mainWrapper}>
+        <form className={styles.form}>
+          <Input
+            placeholder="Enter username"
+            label="Username"
+            inputType="text"
+            id="Username"
```
Comment (marwinburesch) [inline src/app/pages/SignInPage/SignInPage.tsx:43]: `id`s should be lowercase

**Review by marwinburesch — COMMENTED** [review]
👍

**Comment by aronvaupel** [comment]
And why does this page not render????

File: src/app/pages/SignInPage/SignInPage.tsx [hunk 4]
```diff
@@ -12,25 +12,18 @@ function SignInPage(): JSX.Element {
...
-    email.length > 3 ? false : true;
-  };
-
-  const validPassword = () => {
-    password.length > 8 ? false : true;
-  };
+  const validUsername = () => username.length < 3;
+  const validPhonenumber = () => phoneNumber.length < 3;
+  const validEmail = () => email.length < 3;
+  const validPassword = () => password.length < 8;
```
Comment (marwinburesch) [inline src/app/pages/SignInPage/SignInPage.tsx:18]: mmmmh yes
You could add an `is` to the name to make it more apparent what it returns

**Review by marwinburesch — COMMENTED** [review]
`isNice`

**Pull request merged**